indico/modules/events/contributions/util.py

```diff
@@ -221,6 +221,25 @@ def has_contributions_with_user_as_submitter(event, user):
     return _query_contributions_with_user_as_submitter(event, user).has_rows()
 
 
+def get_contributions_for_person(event, person, onlyspeakers=False):
+    """Get all contributions for an event person
+
+    If onlyspeakers is true, then only contributions where the person is speaker
+    are returned
+    """
+    if onlyspeakers:
+        cl_join = db.and_(ContributionPersonLink.is_speaker,
+                          ContributionPersonLink.person_id == person.id,
+                          ContributionPersonLink.contribution_id == Contribution.id)
+    else:
+        cl_join = db.and_(ContributionPersonLink.person_id == person.id,
+                          ContributionPersonLink.contribution_id == Contribution.id)
+
+    return (Contribution.query.filter(Contribution.event_id == event.id)
+            .filter(Contribution.is_deleted is not False)
```

Comment:
that doesn't work, `is` cannot be overridden so this is always calling `.filter(True)`. Use `.filter(~Contribution.is_deleted)` instead.

Reply:
Done